frontend/test/metabase/scenarios/question/filter.cy.spec.js

```diff
@@ -159,4 +160,34 @@ describe("scenarios > question > filter", () => {
       });
     });
   });
+
+  it.skip("should filter based on remapped values (metabase#13235)", () => {
+    withSampleDataset(({ ORDERS, PRODUCTS }) => {
+      // set "Filtering on this field" = "A list of all values"
+      cy.request("PUT", `/api/field/${ORDERS.PRODUCT_ID}`, {
+        has_field_values: "list",
+      });
+      // "Display values" = "Use foreign key" as `Product.Title`
+      cy.request("POST", `/api/field/${ORDERS.PRODUCT_ID}/dimension`, {
+        name: "Product ID",
+        type: "external",
+        human_readable_field_id: PRODUCTS.TITLE,
+      });
+
+      // Add filter as remapped Product ID (Product name)
+      openOrdersTable();
+      cy.findByText("Filter").click();
+      cy.get(".List-item-title")
+        .contains("Product ID")
+        .click();
+      cy.get(".scroll-y")
+        .contains("Aerodynamic Linen Coat")
+        .click();
+      cy.findByText("Add filter").click();
+
+      cy.log("**Reported failing on v0.36.4 and v0.36.5.1**");
+      cy.findByText("Product ID is 7");
```

Comment:
While this work as it is right now, it seems like it might be a P3 regression compared to how things worked pre-33.
`https://github.com/metabase/metabase/issues/8599#issuecomment-468988671`
But let's leave it for now, since I don't know if the "correct" presentation should just be the remapped value (`Product ID is Aerodynamic Linen Coat`) or also include the FK reference (`Product ID is Aerodynamic Linen Coat - 7`) .

Reply:
1. I can completely drop that line and assert only on `subtotal` or some other value from the table
2. I can use regex if filter is at least going to start with "Product ID is 7" (no matter what comes after that)

What do you say?

Reply:
If you want to drop the line, then I'm okay with that, since we got the Subtotal as you say.